kyuubi-common/src/main/scala/org/apache/kyuubi/config/KyuubiConf.scala

```diff
@@ -533,4 +533,11 @@ object KyuubiConf {
     .version("1.2.0")
     .stringConf
     .createWithDefault("SHOW DATABASES")
+
+  val OPERATION_SCHEDULER_POOL: OptionalConfigEntry[String] = buildConf("operation.scheduler.pool")
+    .doc("The scheduler pool of job. Note that, this config should be used after change Spark " +
+      "config spark.scheduler.mode=FAIR.")
+    .version("1.1.1")
```

Comment:
should be 1.2.0?

Reply:
Decide to make 1.1.1 support this minor feature for our requirement : )